### PR TITLE
Update file.cr

### PR DESCRIPTION
**Pull request by rdp**
Add write example doc

### CODE REVIEW

File: src/file.cr [hunk 1]
```diff
@@ -11,13 +11,20 @@ require "crystal/system/file"
...
+# # Implicit close with `open` and a block:
 # content = File.open("path/to/file") do |file|
 #   file.gets_to_end
 # end
 #
-# # Shortcut:
+# # Shortcut of the above:
 # content = File.read("path/to/file")
+# 
+# # Write to a file by opening with a "write mode" specified.
```
Comment (Blacksmoke16) [inline src/file.cr:22]: Might also be worth showing the shortcut equivalent to this like we have for `.read`.

Comment (rdp) [reply src/file.cr:22]: Good idea, why not?

File: src/file.cr [hunk 2]
```diff
@@ -11,13 +11,23 @@ require "crystal/system/file"
...
-# # Shortcut:
+# # Shortcut of the above:
 # content = File.read("path/to/file")
+#
+# # Write to a file by opening with a "write mode" specified.
+# File.open("path/to/file", "w") do |file|
+#   file.print "hello"
+# end
+# # Content of file on disk will now be "hello".
+# See `self.new` for various options *mode* can be.
```
Comment (straight-shoota) [inline src/file.cr:27]:
```suggestion
# # See `.new` for various options *mode* can be.
```

Comment (rdp) [reply src/file.cr]: OK for some reason it wasn't linkable when within a code block either way, so I moved it out and adopted this in feeac26c8, the link works now, thanks for the suggestion.

**Review by straight-shoota — APPROVED** [review]

File: src/file.cr [hunk 3]
```diff
@@ -11,14 +11,24 @@ require "crystal/system/file"
...
+# # Write to a file by opening with a "write mode" specified.
+# File.open("path/to/file", "w") do |file|
+#   file.print "hello"
+# end
+# # Content of file on disk will now be "hello".
+#
+# # Shortcut of the above:
+# File.write("path/to/file", "hello")
 # ```
+# See `new` for various options *mode* can be.
```
Comment (Sija) [inline src/file.cr:31]:
```suggestion
#
# See `new` for various options *mode* can be.
```

**Review by beta-ziliani — APPROVED** [review]
Thanks!

**Pull request merged**